### ISSUE TITLE
Other ways to insert GPS coordinates? FEATURE REQUEST

### ISSUE THREAD

**Issue by TheDestroyer111** [PapaJoesSoup/BDArmory#64]
It just doesn't feel right that only TGP's can give coordinates. Maybe give the GPS coordinator module ability to manually type in the coordinate values? Another way could be by radar, or radar warning receiver.

**Comment by gomker** [comment]
I can see the case for entering coordinates manually.  You can slave your Targeting Pod to Radar and send the coordinates that way - I setup my keypad shortcuts to allow me to quickly set  slave/ send to GPS 

Are you looking to just click the Locked Radar vessel and send to GPS?

**Comment by TheDestroyer111** [comment]
No, not really, not. I especially want the manual input, direct radar could also be cool, but mostly manual input without having to edit gpstargets.cfg is what I want. You could use this to, for example, launch a missile somewhere without ever flying near the target.

**Comment by Toonu** [comment]
Maybe it can use satelite stock KerbNet/SCANsat coordinates for targeting...?

**Comment by gomker** [comment]
@Toonu I like the idea however, each mod can use their own methodology of calculating position (there are a few in the game) I have always wanted to target from space and launch and ICBM :) 
I would like to look into what coordinate systems they use and see if they are easily translated. May have to just pick one to start and see how it goes.

@TheDestroyer111  Good point, I always dislike having to launch a recon aircraft sometimes when I just want to test launch

**Comment by Toonu** [comment]
@gomker Yeah, I know it can be hard to implement even more if this mod is installed then.... but stock KerbNet will be permanent now. Thanks for the efforts!
Unimportant - (I heard RemoteTech will with version 2.0 be overhauled and set on top of CommNet, maybe SCANsat will be later too just over KerbNet instead of its methods to calculate it.)

**Comment by Guard13007** [comment]
Radar warning detecting ground radar should definitely be able to send GPS.

**Comment by Nerfclasher** [comment]
> @gomker Yeah, I know it can be hard to implement even more if this mod is installed then.... but stock KerbNet will be permanent now, so that means, everybody will have it. Thanks for the efforts!
This was talked about during the last final days of me being on the BDAC development team so I won't be surprised if they're working on this as I am typing this and I was actually thinking about this when kerbal space program prerelease for 1.2 I'm pretty sure the team has much more in store yes I know some the plan since I was on the team at the time but I may do the team a favor of keeping you in suspense
